### ISSUE TITLE
Check units in GUI battery parameters

### ISSUE THREAD

**Issue by m-kro** [eclipse-sumo/sumo#15835]
The parameter window contains the entries:

- currently charging [Wh]
- maximum charge rate [W]

Both use the conversion function **MSChargingStation::getChargingPower** from Ws to Wh because the printed values differ only by the charging efficiency factor (see [example](https://github.com/eclipse-sumo/sumo/issues/15815#issuecomment-2507719821)).

**Comment by koharris** [comment]
so, the maximum charge rate is in Ws, not Wh, and the unit currently charged is Ws, not Wh, is that right?

**Comment by m-kro** [comment]
No. Units are fine (if you know that "currently charging" means the energy charged in the last time step). But values and units do not match.

**Comment by koharris** [comment]
"currently charging" means the energy charged in the last time step,
"maximum charge rate" means the max energy can be charged in the last time step.

**Comment by m-kro** [comment]
@koharris:

> "currently charging" means the energy charged in the last time step,
Yes.

> "maximum charge rate" means the max energy can be charged in the last time step.
> The only difference is the charging efficiency.
In order to keep units and values coherent, there is a conversion factor of 3600 (s/h) between the charged energy and the rate. Internally in works like you described (the maximum charge rate is discounted by the charging efficiency and translated into charged energy.
